---
output: 
#  word_document:
  pdf_document:
    keep_tex: true
    fig_caption: true
    latex_engine: pdflatex
    template: ../reference_material/svm-latex-ms.tex
title: "**The effect of a commercial *Ascophyllum nodosum* extracts on tomato and pepper plant productivity and their associated fungal and bacterial communities.**"
author:
- name: Sébastien Renaut^1,2^,Jacynthe Masse^1,2^, Jeffrey P. Norrie^3^, Bachar Blal^^3 Mohamed Hijri^1,2^
  affiliation: ^1^Département de Sciences Biologiques, Institut de Recherche en Biologie Végétale, Université de Montréal, 4101 Sherbrooke Est, Montreal, H1X 2B2, Quebec, Canada. ^2^Quebec Centre for Biodiversity Science, Montreal, Quebec, Canada ^3^Acadian Seaplant Ltd, 30 Brown Avenue, Darthmouth, Nova Scotia, Canaad, B3B 1X8
abstract: "Seaweeds have been used as a source of natural fertilizer and biostimulants for centuries. Here, we used a commercially available *Ascophyllum nodosum* extract (ANE) in order to test its effect of plant productivity in peppers and tomatoes. In addition, by using a metabarcoding high throughput sequencing approach, we characteristed the root and soil fungal and bacterial communities. We find that all productivity measures of root, shoot and fruit biomass differed significantly according to species, and five of those were significantly different according to the fertilization treatment. Local $a$-diversity was the highest in the bacteria-soil and fungi-soil samples, and the lowest in the bacteria-root. In addition, $a$-diversity differed according to the fertilization treatment, but this effect were small. Species composition among sites ($b$-diversity) differed according to the fertilization treatment in all four communities measured (fungal-root, fungal-soil, bacterial-root and bacterial-soil). Finally we identify a number of candidate taxa (Amplicon Sequence Variants) most strongly associated with measures of productivity. Further studies for example using inoculum of microbial species linked to the presence of liquid seaweed extract may help to identify a causative link between extracts, microbes and productivity."
keywords: "Stella Maris®, 16S, ITS, soil microbial diversity, Illumina MiSeq, ANE, Amplicon Sequence Variants, OTU"
date: "`r format(Sys.time(), '%B %d, %Y')`"
geometry: margin=1in
fontfamily: mathpazo
csl: ../reference_material/peerj.csl
fontsize: 11pt
spacing: double
bibliography: ../reference_material/reference.bib
biblio-style: apsr
endnote: no
header-includes:
  - \usepackage[left]{lineno}
  - \linenumbers
---

	 
\newpage 


#INTRODUCTION  
Seaweeds (also known as marine macroalgae) have been used as a source of organic matter and nutrients for centuries, especially in coastal areas [@khan2009seaweed;@craigie2011seaweed]. Liquid seaweed extracts, developped in the 1950s in order to concentrate plant growth-stimulating compounds, facilitate their usage [@milton1952improvements] and today, most commercially available extracts are made from the brown algae *Ascophyllum nodosum*, *Ecklonia maxima* or *Laminaria spp*. One of the main advantages of seaweed extracts is that they are biodegradable, non-toxic and come from a renewable resource, unlike modern chemical fertilizers [@dhargalkar2005seaweed]. Industry-funded bodies such as the European Biostimulant Industry Coalition and the United States Biostimulant Coalition have been working to accommodate biostimulants into mainstream legal architecture. These organizations extoll benefits arising from modes-of-action research, agricultural applications and positive effects on yield and quality of many commercial species (i.e. fruits, vegetables, turf and ornamentals, and woody species). Legal recognition will further allow a fluid integration of various biostimulants, including *Ascophyllum nodosum* Extracts (ANE) into sustainable long-term crop management programs [@duJardin2015; @craigie2011seaweed].  
&nbsp;   
Several comprehensive reviews have described seaweed extracts and their effect on agricultural plant productivity [@khan2009seaweed;@Craigie2010;@craigie2011seaweed;@Battacharyya2015]. The science points to wide-ranging effects from biotic to abiotic resistance, effects on growth and development, and ultimately, to their impact on plant establishment, crop yield and/or quality, and shelf life. At the physiological level, these extracts have been found to influence hormonal changes that in turn, influence physiological processes even at very low concentrations [@wally2013regulation].   
&nbsp;   
Starting in the 1990’s, the development of high quality ANE has led to an increase in cause-effect research, especially on plant diseases [@Jayaraj2015sustainable]. Noted increases in the activity of superoxide dismutase, glutathione peroxidase and ascorbate peroxidase helped support the argument that ANE improve plant tolerance to oxidative stress [@allen2001tasco; @ayad1997effect;@ayad1998effect;@schmidt1997influence]. Positive effects were also found on phytoalexin production [@lizzi1998seaweed;@Jayaraj2008;@Jayaraman2010] suggesting that ANE may be involved in suppressing disease infection through increased activity of these protective enzymes that target oxidizing toxins naturally emitted by disease pathogens.  
&nbsp;   
Improved plant stress resistance and tolerance to foliar and soil treatments is attributed to a cascade of various physiological reactions. ANE can impact plant-signaling mechanisms through a multitude of plant processes and cellular modifications including osmotic/oxidative stresses such as salinity, freezing and drought stress [@jithesh2012analysis]. ANE can also impart drought-stress tolerance to plants by reducing stomatal conductance and cellular electrolyte leakage [Shotton and Martynenko, unpublished data; @spann2011applications]. These results suggest that ANE can influence cellular membrane maintenance leading to a higher tolerance for various osmotic stresses and can mitigate oxidative damage.  
&nbsp;  
Although there is an abundance of published evidence detailing systemic plant effects from ANE, outstanding questions remain as to the effects of ANE on the soil rhizosphere. Various microbes, small arthropods, nematodes, earthworms and insects thriving in the soil rhizosphere help contribute to the aggregation of soil particles, enhance nutrient cycling and delivery to plants, degrade toxic substances, allow better soil water and play a role in plant disease management. An examination of sustainable products that can positively influence microbial interactions between plant roots and soil biota will in turn help to further understand soil borne plant-pathogens competition dynamics. It has been suggested that the plant immune system is composed of inherent surveillance systems that perceive several general microbial elicitors, which allow plants to switch from growth and development into a defense mode [@Newman2013]. This may allow the plant to avoid infection from potentially harmful microbes. The effect of ANE on the bacterial profile suggests that ANE applications increased strawberry root and shoot growth, berry yield and rhizosphere microbial diversity and physiological activity [@alam2013effect]. Similar results were found for sandy loam soils growing carrots as -@Alam2014 showed a strong relationship between carrot growth, soil microbial populations and activity.    
&nbsp;    
The recent development of culture-independent molecular techniques and high throughput sequencing will permit to circumvent the inherent biases of culture-based approaches by targeting the ubiquitous component of life, its DNA. In turn, this should permit to identify a larger proportion of the microbial diversity and lead to a better understanding of the soil microbial response to seaweed extract. DNA barcoding targeting the internal transcribed spacer (ITS) region of the nuclear ribosomal repeat and the bacterial V3-V4 region of the 16S ribosomal gene for fungi and bacteria, respectively, are now regarded as a prerequisite procedure to comprehensively document the diversity and ecology of microbial organisms [@toju2012high;@klindworth2013evaluation].  
&nbsp;  
Here the general objective was to quantify the impact of ANE on plant growth and test how the bacterial and fungal communities responded to the addition of theses extracts. We also aimed to identify specific taxon positively associated with increased in plant productivity following addition of ANE. We hypothesized that the inclusion of liquid seaweed extracts would improve productivity and alter significantly the bacterial and fungal communities. We used a commercially available ANE, Stella Maris®, developped by Acadian Plant Health (NS, Canada). Stella Maris® is derived from the marine algae *A. nodosum*, and harvested from the nutrient-laden waters of the North Atlantic off the Eastern Coast of Canada. We tested the effect of ANE on two commonly used plants (tomato and pepper) using several measures of plant productivity and by measuring soil and root bacterial and fungal diversity using High Throughput Illumina Miseq sequencing.   
&nbsp;   
\newpage  
  
#MATERIAL AND METHOD    
*Experimental design*  
Greenhouse experiments were set up in large trays (60x30x18 cm LxWxH) using two different crops: tomato (*Solanum lycopersicum* L.) and pepper (*Capsicum annuum* L.). Tomato cultivar Totem Hybrid#A371 was planted in November 16th 2015, while pepper cultivar Ace Hybrid#318 was planted in December 9th 2015. Tomato and pepper seeds were purchased from William Dam Seeds Ltd (ON, Canada). These cultivars were selected for greenhouse production. Soil was collected from an agricultural field under organic regime at the IRDA research station in St-Bruno (Qc, Canada, 45^o^32'59.6"N, 73^o^21'08.0"W) on October 7th 2015. The soil was a loamy sand and was collected from the 15 cm top layer. Natural soil was mixed and put into trays, filled to 15 cm in height. Soil analysis was done using a commercial service provided by AgriDirect (Longueuil, QC) and soil characteristics are shown in Table 1. Eight seeds per tray were planted and after germination, only four seedlings per tray were kept.  
&nbsp;  
For each crop species, a randomized split block design (Table S1) was used with four trays set up per block and eight blocks for each experiment. Half of the trays were fertilized (fertilization treatment), as described below. Half of the trays were also planted (planting treatment) with four plants per tray, while the other trays were not planted. This allowed a direct comparison of fungal and bacteria soil communities with respect to fertilization and planting treatments.  
&nbsp;  
Two different fertilization regimes were used according to the plant species. For tomatoes, plants were fertilized using multipurpose organic fertilizer (pure hen manure, 18 g per tray repeated every 4 weeks, 5-3-2) from Acti-sol (Notre-Dame-du-Bon-Conseil, QC) in addition to Stella Maris® (3.5 ml per 1L, each tray received 250 ml, repeated every 2 weeks) for the duration of the experiment. The other half was not fertilized. Stella Maris® is a commercial *Ascophyllum nodosum* seaweed based product and its physical and chemical analyses are shown in Table S2 **WHERE?**. For the pepper experiment, the fertilization regime consisted solely of Stella Maris® (3.5 ml per 1L, each tray received 250 ml, repeated every 2 weeks) for the duration of the experiment. The other half was not fertilized. Both experiments were managed under organic farming practices. Thrips were controlled using *Neoseiulus cucumeris* (syn. *Amblyseius cucumeris*) (1 bag per plant), Fungus gnats were also controlled using predatory mite *Gaeolaelaps gillespiei* (1L; Natural Insect Control, ON). Plants were treated once a week with Milstop, a Potassium Bicarbonate-based foliar fungicide to control the powdery mildew on both crops.    
&nbsp;    
*Plant productivity*  
Tomato and pepper experiments were harvested on March 29th 2016. Measuring the following traits assessed plant productivity: fruit number, fruit weight, shoots fresh weight and roots fresh weight. Traits were measured on three plants chosen randomly per tray for each fertilization-control treatment, crop (tomato/pepper) and block (eight blocks) for a total of 96 samples. In addition, both shoots and roots were dried in a 70 degrees drying oven, and dry weights were quantified after 48 hours. Together, these traits are expected to represent well the plant overall productivity.   
&nbsp;   
&nbsp;  
*Sample preparation, DNA extraction and High throughput sequencing*  
Soil and root samples were taken for both experiments. Soil DNA was extracted using NucleoSpin® Soil DNA extraction kit (Macherey-Nagel, BioLinx, ON) on 250 mg of soil, following the manufacturer’s instructions. Roots were first washed with tap water and rinsed with sterile water. Chopped roots sub-samples (100 mg)  were subjected to DNA extraction using DNeasy Plant Mini kit (Qiagen Inc - Canada, ON), following the manufacturer’s recommendations. Amplicon sequencing targeting bacterial 16S rRNA gene and fungal ITS was performed on both root and soil samples.  
&nbsp;   
For fungal ITS, we used the following primers with the universal CS1 and CS2 adapters: CS1_ITS3_KYO2 (5’-ACA CTGA CGA  CAT GGT TCT ACA GAT GAA GAA CGY AGY RAA-3’) and CS2_ITS4_KYO3 (5’-TAC GGT AGC AGA GAC TTG GTC TCT BTT VCC KCT TCA CTC G-3’) to produce a final amplicon size of approximately 430bp including adapters [@toju2012high].   
&nbsp;  
For bacterial 16S, we used the following primers with CS1 and CS2 universal adapters: 341F (5’-CCT ACG GGN GGC WGC AG-3’) and 805R (5’-GAC TACC AGG GTA TCT AAT C-3’) to produce a final amplicon size of approximately 460 bp and targeting specifically the bacterial V3-V4 region of the 16S ribosomal gene [@klindworth2013evaluation].   
&nbsp;  
DNA samples were then barcoded, pooled and sequenced (2X300bp, paired-end) using an Illumina (San Diego, CA, USA) MiSeq sequencer through a commercial service provided by the Genome Quebec Innovation Centre (Montreal, QC). Sequences were demultiplexed by the sequencing facility and further processed as described below.  
&nbsp;  
*Bioinformatics*  
All bioinformatics, statistical, and graphical analyses further described were performed in R 3.5.1 [@team2018r] and detailed scripts are available here (https://github.com/seb951/Acadian_Seaplants).  
&nbsp;  
We used the `R` package `dada2` [@callahan2016dada2] to infer *Amplicon Sequence Variants* (ASVs). `Dada2` offers accurate sample inference from amplicon data with single-nucleotide resolution in an open source environment. Unlike the Operational Taxonomic Unit (OTU) approach [e.g. @schloss2009introducing; @caporaso2010qiime], ASV are not treated as cluster of sequences defined with an *ad hoc* sequence similarity threshold. Instead, after sequences are quality trimmed and error-corrected, `dada2` reveals the unique members of the sequenced community, thus allowing sequences and abundance counts to be compared among studies [@callahan2016dada2].  
&nbsp;  
First, sequences were trimmed following strict quality thresholds (removing primers and low quality nucleotides, see parameter details in the accompanying `R` scripts). Following this, we applied the error model algorithm of `dada2` which incorporates quality information after filtering, unlike other OTU based methods. Then dereplication, sample inference, merging of paired end reads and removal of chimera were performed in order to obtain a sequence (ASVs) table of abundance per sample. Taxonomy was also assigned using the Ribosomal Database Project (RDP) Naive Bayesian Classifier algorithm from Wang *et al.* [-@wang2007naive]. Depending on support (minimum bootstrap support of 80), we assigned taxonomy from Kingdom to species. We used the silva database formatted for `dada2` to infer bacterial taxa [@silva]. We used the Unite [@UNITE2017] fasta release (including singletons) to infer fungal taxa after formatting it to the `dada2` format using a custom `R` script. The pipeline was run on a multithreaded (48 CPUs) computer infrastructure provided by Westgrid (https://www.westgrid.ca/support/systems/cedar) and Compute Canada (www.computecanada.ca). Note that the pipeline was run separately for fungal-root, fungal-soil, bacteria-soil and bacteria-root samples given the markedly different nucleotide compositions of the sequenced amplicons, unique taxa and specific error models of each dataset.
&nbsp;  
&nbsp;  
*Statistical analyses - plant productivity*  
We tested for the effect of species (tomato vs pepper), fertilization and their interaction on six plant productivity measures (fruit number, average fruit weight, shoots fresh weight, roots fresh weight, shoots dry weight, roots dry weight). We used linear mixed effect models (LMM) in the R package `nlme` [@pinheiro2017nlme], which are more appropriate than an Analysis of Variance (ANOVA) given the current block design (blocks and replicates nested within a block were treated as random variables). All six plant productivity measures were either square root or log transformed in order to help satisfy the assumption of normality of the residuals in the LMM statistical framework. For the variables *fruit number* and *average fruit weight*, we also used a permutation-based 2-way ANOVA [@anderson1999empirical] given that the residuals of the LMM were not normally distributed (results were similarly significant).      
&nbsp;  
*Statistical analyses - microbial and fungal diversity*  
Fungal-root, fungal-soil, bacterial-root and bacterial-soil ASV diversity was measured separately. For each of these four datasets, we removed samples that showed poor sequencing output and contained few ASVs. In order to do this, we summed the abundance of all ASVs for each sample ($\sum_{i=1}^n ASV$) and eliminated samples that had fewer that the mean sum minus four standard deviations ($\overline{\sum_{i=1}^n ASV} - 4\sigma$). In addition, we removed ASVs from our dataset that were present in fewer than 5% of the samples (less than ten individuals in the soil samples or less than five in the root samples). This was done to remove very rare ASVs unique to a block or replicate, but not found in the majority of samples.  
&nbsp;    
We then conducted community-based analyses looking at the effect of the fertilization treatment on the ASV taxa in the tomato and pepper experiments. To reduce the complexity of the datasets, relative abundance of all taxa was calculated per family using the `R` package `dplyr` [@wickham2015dplyr]. Barplots were drawn using `ggplot2` [@wickham2016ggplot2] to visualize communities. ASV alpha ($a$)-diversity was calculated for each sample using the inverse Simpson diversity index in `vegan` [@oksanen2013package]. The effect of the fertilization treatment, species (and planting for soil communities) were assessed using a linear mixed-effect (LMM) model in the R package `nlme` [@pinheiro2017nlme], given the unbalanced, replicated block design. Alpha diversity was log transformed in order to help satisfy the assumption of normality of the residuals of the LMM statistical framework.  
&nbsp;  
Using the community matrix data of ASVs abundance, we performed PERmutational Multivariate ANalysis Of VAriance tests [PERMANOVA; @anderson2001new] to identify relationships between the communities according to the experimental design. ASV abundance data was Hellinger-transformed and significance was assessed using 10,000 permutations in `vegan` [@oksanen2013package]. Blocks and replicates nested within blocks were factored as strata (blocks) in the model.  
&nbsp;    
We also performed canonical correspondence analyses (CCAs) using Hellinger-transformed ASV abundance data in `vegan` [@oksanen2013package] to visually assess the grouping of samples, ASVs and their association with productivity variables (*species* scaling based on ASV matrix). Data were analyzed separately for fungal-root, fungal-soil, bacterial-root and bacterial-soil, but also according to species (tomato/pepper), given that analyses of $a$-diversity showed that tomato and pepper were markedly different. This gave a total of eight CCAs. Data were constrained based on four productivity measures (fruit number, average fruits weight, shoots fresh weight, roots fresh weight). We excluded the shoot & root dry weights as constraints to simplify the model. In addition, these were highly correlated with the fresh weight already included as constraints ($r^2$=0.98 and 0.76 for shoot dry/fresh weights and root dry/fresh weights, respectively).      
&nbsp;    
Finally, we attempted to identify candidate ASVs positively associated with productivity. As such, we  identified the ten ASVs most positively associated with the measures of fruit number, shoots fresh weight and roots fresh weight from each canonical correspondence analysis for a total of 40 fungal and 40 bacterial candidate ASVs. We aligned candidate sequences from these candidates ASVs using the Bioconductor `R` package `decipher` [@wright2016using] and build pairwise distances matrices using a JC69 substitution models of DNA sequence evolution (equal base frequencies, [@jukes1969evolution] in `phangorn` [@schliep2010phangorn]. Phylogenetic trees for bacteria and fungi were plotted using `ape` [@paradis2004ape]. This permitted to identify if similar candidate ASVs were found under different experimental conditions (soil/root, pepper/tomato), thus reinforcing their role in productivity increase, and decreasing the false positive rate.  
&nbsp;  
  
\newpage  

#RESULTS  
  
*Soil characteristics*  
In Table 1, we present the characteristics of the soil which was collected at the IRDA research station in St-Bruno (Qc, Canada) and used in the current experimental design.  
&nbsp;    
```{r soilcharac, echo=F}
library(knitr)
permanova.summary = read.table("../results/nutrients.csv",sep = ",", header = T)
colnames(permanova.summary) = sub("."," ",colnames(permanova.summary),fixed =T)
kable(permanova.summary,digits =4,caption = "Soil characteristics (in ppm unless specified otherwise)")
```  

&nbsp;  
&nbsp;  
*Plant productivity*  
The effects of the fertilization treatment were tested on overall plant growth and six measures of productivity (i.e. fruit number, average fruit weight, shoots fresh weight, shoots dry weight, roots fresh weight, roots dry weight) for both tomato and peppers. Visually, both above ground and below ground plant structure grew larger in fertilized tomato (hen manure + ANE) and pepper plants (ANE only), in addition to producing more fruits (see Figure 1 for some examples of the differences between fertilized and unfertilized plants). 
&nbsp;  
![Figure 1](../figures/Figure_2_photos_productivity.png){width=500px}  
**Figure 1: Plant productivity. Photos were taken at the end of the experimental treatment. In each photo, fertilized plants are on the left. A: pepper plants, B: pepper roots, C: pepper fruits and D: tomato fruits.**  
&nbsp;  
Statistically, all six productivity measures differed significantly according to species, and five of those were significantly different according to the fertilization treatment (Figure 2). The fertilization effect was stronger in the tomato plants (see fold changes in Figure 2), likely due to the fact that these plants were fertilized with both hen manure and ANE. The only exception was the average fruit weight that did not differ between fertilized and control plants (LMM, $F_{(1,69)}$ = 1.27, *p*-value=0.26). However, the model did reveal a significant interaction between treatment and plant ($F_{(1,69)}$ = 9.6, *p*-value=0.0028). In fact, when testing only the pepper plants, the effect of fertilization on average fruit weight was significantly higher in the fertilized pepper plants ($F_{(1,23)}$ = 10.84, *p*-value=0.0032).  
&nbsp;  
![Figure 2](../figures/Figure_3_productivity.pdf){width=600px}  
**Figure 2: Measures of plant productivity. *a* and *b* subscripts above boxplots denote significant differences. Fold changes between the mean of the control and fertilized plants were also noted for significant changes (for pepper and tomato separately).**  
&nbsp;  
&nbsp;  
*Sequencing*  
A total of 2.7 million paired-end raw reads were obtained for all samples combined (976,000 for fungi-soil, 920,000 for fungi-root, 309,000 for bacteria-soil and 535,000 for bacteria-root, Table 2). Note that sequencing samples were analyzed separately for fungal-soil, fungal-root, bacteria-soil and bacteria-root conditions. On average, 47,664 paired-end reads were obtained per sample. After quality filters were applied, including removing chimeras, and paired-end reads were merged, an average of 19,690 sequences remained per sample. From 192 soil samples for fungi and bacteria, and 96 root samples for fungi and bacteria sequenced, seven fungi-soil samples, 15 fungi-root samples and one bacteria-root samples were removed because they had to few reads based on our strict quality thresholds.  
&nbsp;  
The `dada2` pipeline inferred, on average, 170 Amplicon Sequence Variants per sample (average of 176 fungal-soil ASV, 37 fungal-root ASVs, 269 bacterial-soil ASVs and 92 bacterial-root ASVs). Many of those were unique to one of a few samples (total number of 6,112 fungal-soil, 845 fungal-root, 9,352 bacterial-soil and 2,023 bacterial-roots ASVs). After quality filtering ASVs found in fewer than 10% of the samples, we retained 413, 106, 811 and 325 ASVs. These retained ASVs comprised 94%, 95%, 89% and 98% of all reads in the fungal-soil, fungal-root, bacterial-soil and bacterial-root samples, respectively.  
&nbsp;  
&nbsp; 
```{r summary Table2, echo=F}
library(knitr)
#library(kableExtra)
summary.table=read.table("../results/summary.table",header = T,row.names = 1)
colnames(summary.table)[2] = "Nb_seq_sum"
summary.table[,2] = summary.table[,2]*1000
summary.table=t(summary.table[c(4:1),])
summary.table[1,] = c(192,96,192,96)
summary.table = summary.table[c(2:6,1,9,8,10,7),]
rownames(summary.table) = c("No sequences (sum)","No sequences (mean)","No seq. filtered (mean)","No seq. filt. merged (mean)","No seq. filt. merg. no chimeras (mean)","No samples","No samples trimmed","No ASVs (sum)","No ASVs trimmed (sum)","ASV per sample (mean)")
kable(summary.table,format.args=list(big.mark = ','), digits =4,caption = "Sequencing and ASV summary")
```  

&nbsp;  
*Root, soil, microbial and bacterial diversity*  
The entire community structure that was measurable in the soil was then analyzed and the relative abundance of taxa (family) for the fungal-soil, fungal-root, bacteria-soil and bacteria-root conditions was reported (Figure 3a & b). Fungal communities were dominated by Nectriaceae, both in the root and soil samples. The bacterial family Bacilaceae dominated to a lesser extent the soil communities. Bacterial root communities were largely dominated by the Cyanobacteria phylum (identified as *chloroplast* in the silva database according to the RDP Bayesian Classifier).     
&nbsp;  
![Figure 3a](../figures/Figure4_FAMILY_barplots_fungi.pdf){width=700px}  
**Figure 3a: Barplots of the relative abundance of fungal ASVs for fungi**  
&nbsp;
&nbsp;  
![Figure 3b](../figures/Figure4_FAMILY_barplots_bacteria.pdf){width=700px}  
**Figure 3b: Barplots of the relative abundance of bacterial ASVs for bacteria**  
&nbsp;
&nbsp;  
*Local ($a$-diversity)*  
The diversity of each site ($a$-diversity) was calculated separately for each sample and under each experimental condition (fungi-soil, fungi-root, bacteria-soil and bacteria-root, Figure 4). Total $a$-diversity was the highest in the bacteria-soil and fungi-soil samples, and the lowest in the bacteria-root. Linear mixed effects models were used to assess significance. In soil samples, fungal diversity did not differ with respect to the fertilization ($F_{(1,161)}$=0.17, *p*-value=0.6853), but did so with respect to planting ($F_{(1,161)}$=9.00, *p*-value<0.0032) and species ($F_{(1,161)}$=13.03, *p*-value=0.0003) treatments. In root samples, fungal diversity differed with respect to the fertilization treatment ($F_{(1,56)}$=10.1, *p*-value=0.003), and the species tested ($F_{(1,56)}$=4.5, *p*-value=0.04). In soil samples, bacterial diversity differed with respect to the fertilization ($F_{(1,165)}$=17.13, *p*-value<0.0001), planting ($F_{(1,165)}$=139.0, *p*-value<0.0001) but not species ($F_{(1,165)}$=1.89, *p*-value=0.17) treatments. In root samples, bacterial diversity differed with respect to the fertilization treatment ($F_{(1,67)}$=17.27, *p*-value=0.0001), and the species tested ($F_{(1,67)}$=359.69, *p*-value<0.0001).
&nbsp;  
![Figure 4](../figures/Figure5_alpha.pdf){width=600px}  
**Figure 4: Boxplot of alpha diversity according to the treatment, species and planting effect for fungal-root, fungal-soil, bacteria-soil and bacteria-root. *a* and *b* subscripts above boxplots denote significant differences.**  
&nbsp;  
&nbsp;  
*Differences in species composition among sites*  
Using a PERMANOVA statistical framework, we identified that for all conditions, communities differed with respect to the fertilization treatment (Table 3). Soil fungal and bacterial communities differed the most according to whether the tray was planted (greatest % of variance explained by factor, Table 3), while root communities differed most with respect to the species (tomato/pepper) factor.  
  
```{r permanova Table3, echo=F}
library(knitr)
permanova.summary = read.table("../results/permanova.summary")
colnames(permanova.summary) = c("fungi-soil","fungi-root","bacteria-soil","bacteria-root")
kable(permanova.summary,digits =4,caption = "summary of the factors tested in the PERMANOVAs ($r^2$ and *p*-values)*")
```  
  
\*$r^2$ (percentage of variance explained by the term in the model) and associated *p*-values in parentheses.
&nbsp;   
&nbsp;   
*Canonical correspondence analyses and candidate ASVs*  
Canonical correspondence analyses (CCAs) indicated how fertilized samples clustered together according to their fungal or bacterial communities (Figure 6). It also shows a similar association of three of the constrain variables (productivity measures of root fresh weight, shoots fresh weight and fruit number), while average fruit weight behave differentially as noted previously in Figure 2 (in fact nearly orthogonally to the other three constrains in most ordinations).  
&nbsp;  
\newpage  
  ![Figure 5](../figures/Figure6_rda.pdf){width=400px, dpi=700}  
**Figure 6: Canonical correspondence analyses for tomato (A-D) and peppers (E-H) for soil-fungi, root-fungi, soil-bacteria and root-bacteria. Samples were labeled and colored in gray (unfertilized) or dark yellow (fertilized). Red crosses represent individual ASVs, while red points represent the ten ASVs most closely associated with the three productivity measures of root fresh weight, shoots fresh weight and fruit number. Blue arrows are the four productivity measures used as constrains in the ordinations.** 
&nbsp;  
&nbsp;  
Next, we identified, for each ordination, the ten ASVs most closely related to the three constrains of root fresh weight, shoots fresh weight and fruit number. These ASVs were considered as putative candidate sequences most positively impacted (increase presence of the ASV) by fertilization. We further analyzed the corresponding sequences for these eighty candidate ASVs (ten candidates \* eight ordinations)in two separate alignments (one for fungi and one for bacterial ASVs) and their accompanying phylogenetic trees. In fungi, we identified one cluster of ASVs taxonomically assigned to *Mortierella* (soil saprotrophs in the phylum Zygomycota) positively associated to productivity in both tomato and pepper roots. In addition, we identified a cluster of four different fungal ASVs in tomato soil (ASV132, ASV153) and pepper-root (ASV19 & AV17) closely related phylogenetically. Given that no taxonomy was assigned to these sequences through the `dada2` RDP bootstrap approach, we used a BLASTn [@altschul1997gapped] approach (against NCBI nr) to identify the most closely related sequences. We identified this cluster of ASVs as *Rhogostoma schuessleri (BLASTn, e-value=4e-76)*, a protist in the phylum Cercozoa, which is known to be present in the rhizo and phyllo-sphere [@dumack2017rhogostomidae]. A number of putative plant pathogenic fungi were also identified such as *Fusarium sp.*, *Microdochium colombiense* or *Setophoma terrestris*.
&nbsp;  
In bacteria-roots, we identified a large number of different ASVs most positively impacted (increase presence of the ASV) by fertilization (Figure 6).
&nbsp;  
&nbsp;  
![Figure 6](../figures/Figure7_candidateASVs.pdf){width=400px, dpi=500}  
**Figure 6: Neighbor-Joining trees of candidates ASVs (fungi & roots) associated with productivity measures. The most accurate taxonomy assigned according to the RDP bayesian classifier (form Phylum to species) was added as tip labels.** 
&nbsp;  
&nbsp;  

\newpage  
  
#DISCUSSION  
In the current study, we investigated the effects of *Ascophyllum nodosum* Extracts (ANE) on root and shoot characteristics in addition to fruit yield and bacterial and fungal communities in tomato and pepper. Overall measures of root, shoot and fruit productivity increased for both species following the addition of ANE. As such, our results corrobate previous studies documenting the impact of ANE on productivity in strawberries [@alam2013effect] and carrots [@Alam2014].   
&nbsp;   
In the tomato experimental set up, the effect of fertilization was especially high, liley due to the fact that plants were also fertilized with hen manure (5-3-2, 18g per tray / 2 weeks) in addition to ANE (see Figure 2). This was not the case for the pepper plants and the increase in productivity was solely due to the addition of ANE. The commercial extract (Stella Maris®, Acadian Seaplants Ltd ) used contained about 1% nitrogen, 0.5% phosphorus, 15% potassium, 0.4% calcium, 0.4% magnesium, 155 ppm iron, 121 ppm manganese, 5 ppm copper, 91 ppm zinc, and 124 ppm boron. In the current experimental setup, ANE was diluted to 3.5 g / L prior to application (250ml per tray / 2 weeks). As such, these nutrients were given at very low concentrations relative to the crop requirements and are not expected to significantly impact growth relative to a regular agricultural fertility program [@alam2013effect]. In fact, the amounts of Nitrogen and Phosphorus supplied via the application of ANE were ~100 less than from the hen manure itself in the tomato plants. Instead, bioactive compounds such as betaines, polyamines, cytokinins, auxins, oligosaccharides, amino acids and vitamins have been found to have many overall beneficial productivity effects on plant growth [@khan2009seaweed;@Craigie2010;@craigie2011seaweed;@Battacharyya2015]  
&nbsp;  
Here, one of primary goal of the study was to document how the bacterial and fungal communities responded to the addition of ANE. We used a metabarcoding high throughput sequencing approach targeting a DNA region specific to all fungi (ITS) and bacteria (16S). Then, we identified bacterial and fungal communities using a relatively novel bioinformatics approach developed by -@callahan2016dada2. The approach, based on the widely used programming language `R` [@team2018r], identifies unique, non-clustered, sequences (ASVs) that are then comparable among studies. In the current study, most ASVs identified were rare and unique to one or a few sample. In fact, ~90% of all ASVs were discarded given that they were found in very few samples and were thus not representative of a particular experimental treatment. Yet, these ASVs comprised a small minority of all sequencing reads (~5% of all sequences). In addition, the current analytical pipeline used a bayesian classifier approach to taxonomy rather than the widely used BLAST approach, thus providing more conservative, but more accurate taxonomic identifications [@wang2007naive].     
&nbsp;  
The total number of ASVs per site ($a$-diversity) for bacteria and fungi differed with respect to the fertilization treatment in root samples and the soil (only for bacteria), but these effects were small (Figure 5). Nectriaceae, a family of fungi in the order Hypocreales and often encountered as saprophytes on decaying organic matter comprised most of the diversity both in the soil on root of plants (between 25-70% of the total number of sequencing reads, Figure 4a). With respect to soil-bacteria, communities were much more diverse and comprised many different family (Figure 4b). Surprisingly, most sequencing reads in the root-bacteria communities likely originate from the plants themselves (identified as chloroplastic or mitochondrial in origin in Figure 4b), despite the fact that the DNA primer pair used should have primarily targeted the bacterial V3-V4 region of the 16S ribosomal gene.  
&nbsp;    
Species composition among sites ($b$-diversity) differed according to the fertilization treatment in all four communities measured (fungal-root, fungal-soil, bacterial-root and bacterial-soil). This fertilization effect was small (2-7% of variance explained in the models, Table 3), but significant. Most of the variance was explained by the planting treatment in the soil and the species (tomato/pepper) in the roots (Table 3).   
&nbsp;  
We also aimed to identify specific taxon positively associated with increased in plant productivity following addition of ANE. Here we discuss some of the candidates.  In fungi, we identified one cluster of ASVs taxonomically assigned to *Mortierella* (soil saprotrophs in the phylum Zygomycota) positively associated to productivity in both tomato and pepper roots. In addition, we identified several fungal ASVs in tomato soil and pepper-root linked to productivity. These were assigned to *Rhogostoma schuessleri (BLASTn, e-value=4e-76)*, a protist in the phylum Cercozoa, which is known to be present in the rhizo and phyllo-sphere [@dumack2017rhogostomidae]. A number of putative plant pathogenic fungi were also identified such as *Fusarium sp.*, *Microdochium colombiense* or *Setophoma terrestris*. In bacteria-roots, we identified a large number of different ASVs most positively impacted (increase presence of the ASV) by fertilization (Figure 6).  
&nbsp;  
DNA barcoding is now regarded as a prerequisite procedure to comprehensively document the diversity and ecology of microbial organisms [@toju2012high;@klindworth2013evaluation]. Further studies for example using inoculum of microbial species linked to the presence of liquid seaweed extract may help to identify a causative link between extracts, microbes and productivity.  
&nbsp;  
&nbsp;  

#ACKNOWLEDGMENTS
We thank Mengxuan Kong for technical assistance in raising the plants and measuring productivity and Simon Morvan for discussion about bioinformatics analyses and seaweed extracts.
  
\newpage  
  
#REFERENCES
&nbsp;  