---
output: 
#  word_document:
  pdf_document:
    keep_tex: false
    fig_caption: true
    latex_engine: pdflatex
    template: ../bibliography/sr-latex-ms.tex
title: "**A commercial seaweed extract structured microbial communities associated with tomato and pepper roots and significantly increased crop yield**"
author:
- name: Sébastien Renaut^1,2^,Jacynthe Masse^1,2^, Jeffrey P. Norrie^3^, Bachar Blal^3^ Mohamed Hijri^1,2^
  affiliation: ^1^Département de Sciences Biologiques, Institut de Recherche en Biologie Végétale, Université de Montréal, 4101 Sherbrooke Est, Montreal, H1X 2B2, Quebec, Canada. ^2^Quebec Centre for Biodiversity Science, Montreal, Quebec, Canada ^3^Acadian Seaplant Ltd, 30 Brown Avenue, Darthmouth, Nova Scotia, Canada, B3B 1X8  
abstract: "Seaweeds have been used as a source of natural fertilizer and biostimulant in agriculture for centuries. However, their effects on soil and crop roots microbiota remain unclear. Here, we used a commercially available *Ascophyllum nodosum* Extract (ANE) to test its effect on bacterial and fungal communities of rhizospheric soils and roots of pepper and tomato plants in greenhouse trials. Two independent trials were conducted in a split block design to test to effect of ANE amendment. We used amplicon sequencing targeting fungal ITS and bacterial 16S rRNA gene to determine microbial community structure changes. We find that productivity parameters of root, shoot and fruit biomass were positively and significantly influenced by the amendment. In addition, $a$-diversity differed significantly between amended and control plants, but only in some of the experimental conditions. Species composition among sites ($b$-diversity) differed according to the amendment treatment in all four communities (fungal-root, fungal-soil, bacterial-root and bacterial-soil). Finally, we identified a number of candidate taxa most strongly correlated with crop yield increases. Further studies on isolation and characterization of these microbial taxa linked to the application of liquid seaweed extract may help to enhance crop yield in sustainable agro-ecosystems."
runhead: "Seaweed extracts affect microbial communities"
keywords: "Stella Maris®, 16S, ITS, soil microbial diversity, Illumina MiSeq, ANE, Amplicon Sequence Variants, OTU"
date: "`r format(Sys.time(), '%B %d, %Y')`"
geometry: margin=1in
fontfamily: mathpazo
csl: ../bibliography/envmicrob.csl
fontsize: 12pt
spacing: double
bibliography: ../bibliography/reference.bib
biblio-style: apsr
endnote: no
header-includes:
  - \usepackage[left]{lineno}
  - \linenumbers
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE,eval=F)
```
	 
\newpage 


# SIGNIFICANCE STATEMENT
While seaweeds have been used as a source of natural fertilizer and biostimulant for centuries, their effects on soil microbiota remain unclear. Here, we showed that plant productivity was positively and significantly influenced by the addition of a liquid *Ascophyllum nodosum* seaweed extract. Then, we used amplicon sequencing targeting fungal ITS and bacterial 16S rRNA gene to quantify Amplicon Sequence Variants and changes in the microbial community structure. Both bacterial and fungal species composition differed according to the amendment treatment and we identified a number of candidate taxa whose presence was correlated with plant yield increases. As such, these promising microbial candidates linked to the application of liquid seaweed extract may help to enhance crop yield in sustainable agro-ecosystems.


\newpage 

# INTRODUCTION    
Seaweeds (also known as marine macroalgae) have been used as a source of organic matter and mineral nutrients for centuries, especially in coastal areas [@khan2009seaweed;@craigie2011seaweed]. Liquid seaweed extracts, developed in the 1950s in order to concentrate plant growth-stimulating compounds, facilitate their usage [@milton1952improvements]. Today, most commercially available extracts are made from the brown algae *Ascophyllum nodosum*, *Ecklonia maxima* or *Laminaria spp*. Unlike modern chemical fertilizers, seaweed extracts are biodegradable, non-toxic and come from a renewable resource [@dhargalkar2005seaweed]. Industry-funded bodies such as the European Biostimulant Industry Coalition and the United States Biostimulant Coalition have been working to accommodate biostimulants into mainstream legal architecture. These organizations extoll benefits arising from modes-of-action research, agricultural applications and positive effects on yield and quality of many commercial species (i.e. fruits, vegetables, turf, ornamentals and woody species). Legal recognition will further allow a better integration of various biostimulants, including *Ascophyllum nodosum* Extracts (ANE) into sustainable long-term crop management programs [@duJardin2015; @craigie2011seaweed].  
&nbsp;   
Starting in the 1990’s, high quality ANE was developed and let to an increased usage by farmers, in addition to an increase in cause-effect research, especially on plant diseases [@Jayaraj2015sustainable]. Since then, several comprehensive reviews have described the effects of seaweed extracts on agricultural plant productivity [@khan2009seaweed;@Craigie2010;@craigie2011seaweed;@Battacharyya2015]. The science points to wide-ranging effects from biotic to abiotic resistance, effects on growth and development, and ultimately, to their impact on plant establishment, crop yield and/or quality, and shelf life.  
&nbsp;   
At the physiological level, these extracts have been found to influence hormones levels that in turn, influence physiological processes even at very low concentrations [@wally2013regulation]. Noted increases in the activity of superoxide dismutase, glutathione peroxidase and ascorbate peroxidase helped support the argument that ANE improve plant tolerance to oxidative stress [@allen2001tasco;@ayad1997effect;@ayad1998effect;@schmidt1997influence]. Positive effects were also found on phytoalexin production suggesting that ANE may increase activity of these protective enzymes that target oxidizing toxins naturally emitted by disease pathogens [@lizzi1998seaweed;@Jayaraj2008;@Jayaraman2010].  
&nbsp;   
Improved plant stress resistance and tolerance to foliar and soil pest treatments is attributed to a cascade of various physiological reactions. ANE can impact plant-signaling mechanisms through a multitude of plant processes and cellular modifications including osmotic/oxidative stresses such as salinity, freezing and drought stress [@jithesh2012analysis]. ANE can also impart drought-stress tolerance to plants by reducing stomatal conductance and cellular electrolyte leakage [Shotton and Martynenko, unpublished data; @spann2011applications]. These results indicate that ANE can influence cellular membrane maintenance leading to a higher tolerance for various osmotic stresses and can mitigate oxidative damage.  
&nbsp;  
Although published evidence has detailed systemic effects of ANE on plant development, the effect of ANE on the biology of the rhizosphere is still largely unknown. The soil rhizosphere harbors a large microbial biodiversity that contributes to the aggregation of particles, enhances nutrient cycling and delivery to plants, degrades toxic substances, allows better soil water retention and plays a role in plant disease management. It has been suggested that the plant immune system is composed of inherent surveillance systems that perceive several general microbial elicitors, which allow plants to switch from growth and development into a defense mode [@Newman2013]. This may allow the plant to avoid infection from potentially harmful microbes. An examination of sustainable products that can positively influence microbial interactions between plant roots and soil biota will in turn help to further understand soil borne plant-pathogens competition dynamics. The effect of ANE on the bacterial profile suggests that ANE applications increased strawberry root and shoot growth, berry yield, and rhizosphere microbial diversity and physiological activity [@alam2013effect]. Similar results were found in sandy loam soils as Alam and colleagues -@Alam2014 showed a strong relationship between carrot growth, soil microbial populations and activity.    
&nbsp;    
The recent development of culture-independent molecular techniques and high throughput sequencing should permit to circumvent the inherent biases of culture-based approaches by targeting the ubiquitous component of life, DNA. In turn, this will help to identify a larger proportion of the microbial diversity and lead to a better understanding of the soil microbial response to seaweed extract. DNA barcoding targeting specific regions of the genome (e.g. ITS: fungi, 16s ribosomal genes: bacteria) are now regarded as a prerequisite procedure to comprehensively document the diversity and ecology of microbial organisms [@toju2012high;@klindworth2013evaluation].  
&nbsp;  
Here, the general objective was to quantify the impact of a commercial seaweed extract on plant growth and test how the fungal and bacterial communities responded to the addition of these extracts. We also aimed to identify specific taxon positively correlated with increases in plant productivity following ANE amendments. We hypothesized that the addition of liquid seaweed extracts would improve productivity and alter significantly the fungal and bacterial communities. We used a commercially available ANE, Stella Maris®, developed by Acadian Seaplants Ltd (NS, Canada). Stella Maris® is derived from the marine algae *A. nodosum*, and harvested from the nutrient-laden waters of the North Atlantic off the Eastern Coast of Canada. We tested the effect of ANE amendment on two agricultural plants commonly grown in greenhouse conditions (tomato and pepper). Several traits related to plant productivity were measured and soil and root bacterial and fungal diversity were quantified using High Throughput Illumina (San Diego, CA, USA) Miseq sequencing.   
&nbsp;   
\newpage  
  
# MATERIAL AND METHOD      
*Experimental design*  
Greenhouse trials were set up in large trays (60x30x18 cm LxWxH) using two different crops: tomato (*Solanum lycopersicum* L.) and pepper (*Capsicum annuum* L.). Tomato cultivar Totem Hybrid#A371 was planted in November 16th 2015 and pepper cultivar Ace Hybrid#318 was planted in December 9th 2015. Tomato and pepper seeds were purchased from William Dam Seeds Ltd (ON, Canada). These cultivars were selected for greenhouse production. Soil was collected from an agricultural field under organic regime at the IRDA research station in St-Bruno (Qc, Canada, 45^o^32'59.6"N, 73^o^21'08.0"W) on October 7th 2015. The soil was a loamy sand and was collected from the 15 cm top layer. Natural soil was homogenized and put into trays, filled to 15 cm in height. Soil analysis was done using a commercial service provided by EnvironeX (formerly AgriDirect, Longueuil, QC) and soil characteristics are shown in Table S1. Eight seeds per tray were planted and after germination, only four seedlings per tray were kept.  
&nbsp;  
For each plant species, a randomized split block design (Table S2) was used with four trays set up per block and eight blocks for each trial. Half of the trays were amended with ANE, as described below. Half of the trays were also planted (planting effect) with four plants per tray, while the other trays were not planted. This allowed a direct comparison of fungal and bacteria soil communities with respect to the amendment and planting effects.  
&nbsp;  
Two different amendment regimes were used according to the plant species. For tomatoes, plants were amended using multipurpose organic fertilizer (pure hen manure, 18 g per tray repeated every 4 weeks, 5-3-2) from Acti-sol (Notre-Dame-du-Bon-Conseil, QC) in addition to Stella Maris® (3.5 ml per 1L, each tray received 250 ml, repeated every 2 weeks) for the duration of the experiment. The other half were not treated, but watered with 250ml per tray instead. The physico-chemical composition of Stella Maris® is shown in Table S3. For the pepper experiment, the amendment treatment consisted solely of Stella Maris® (3.5 ml per 1L, each tray received 250 ml, repeated every 2 weeks) for the duration of the experiment. The other half was not amended, but watered with 250 ml per tray instead. Both experiments were managed under organic farming practices. Thrips were controlled using *Neoseiulus cucumeris* (syn. *Amblyseius cucumeris*) (1 bag per plant), Fungus gnats were also controlled using predatory mite *Gaeolaelaps gillespiei* (1L; Natural Insect Control, ON). Plants were treated once a week with Milstop, a Potassium Bicarbonate-based foliar fungicide to control the powdery mildew on both crops.    
&nbsp;    
*Plant productivity*  
Tomato and pepper experiments were harvested on March 29th 2016. The following traits assessed plant productivity: fruit number, fruit weight, shoots fresh weight and roots fresh weight. Together, these traits are expected to represent well the plant overall productivity. Traits were measured on three plants chosen randomly per tray for each amended / control plant, crop (tomato / pepper) and block (eight blocks) for a total of 96 samples. In addition, both shoot and root samples were dried in a 70 degrees drying oven, and dry weights were quantified after 48 hours.     
&nbsp;   
*Sample preparation, DNA extraction and High throughput sequencing*  
Soil and root samples were taken for both experiments. Soil DNA was extracted using NucleoSpin® Soil DNA extraction kit (Macherey-Nagel, BioLinx, ON) on 250 mg of soil, following the manufacturer’s protocol. Roots were first washed with tap water and rinsed with sterile water. Chopped roots sub-samples (100 mg) were subjected to DNA extraction using DNeasy Plant Mini kit (Qiagen Inc - Canada, ON), following the manufacturer’s recommendations. Amplicon sequencing targeting bacterial 16S rRNA gene and fungal ITS was performed on both root and soil samples.  
&nbsp;   
For fungal ITS, we used the following primers with the universal CS1 and CS2 adapters: CS1_ITS3_KYO2 (5’-ACA CTG ACG ACA TGG TTC TAC AGA TGA AGA ACG YAG YRA A-3’) and CS2_ITS4_KYO3 (5’-TAC GGT AGC AGA GAC TTG GTC TCT BTT VCC KCT TCA CTC G-3’) to produce a final amplicon size of approximately 430bp including adapters [@toju2012high].   
&nbsp;  
For bacterial 16S, we used the following primers with CS1 and CS2 universal adapters: 341F (5’-CCT ACG GGN GGC WGC AG-3’) and 805R (5’-GAC TAC CAG GGT ATC TAA TC-3’) to produce a final amplicon size of approximately 460 bp and targeting specifically the bacterial V3-V4 region of the 16S ribosomal gene [@klindworth2013evaluation].   
&nbsp;  
DNA samples were then barcoded, pooled and sequenced (2X300bp, paired-end) using an Illumina MiSeq sequencer through a commercial service provided by the Genome Quebec Innovation Centre (Montreal, QC). Sequences were demultiplexed by the sequencing facility and further processed as described below.  
&nbsp;  
*Bioinformatics*  
All bioinformatics, statistical, and graphical analyses further described were performed in R 3.5.1 [@team2018r] and detailed scripts are available here (https://github.com/seb951/Acadian_Seaplants).  
&nbsp;  
We used the `R` package `DADA2` [@callahan2016dada2] to infer *Amplicon Sequence Variants* (ASV). `DADA2` offers accurate sample inference from amplicon data with single-nucleotide resolution in an open source environment. Unlike the Operational Taxonomic Unit (OTU) approach [e.g. @schloss2009introducing; @caporaso2010qiime], ASV are not treated as cluster of sequences defined with an *ad hoc* sequence similarity threshold. Instead, after sequences are quality trimmed and error-corrected, `DADA2` reveals the unique members of the sequenced community, thus allowing sequences and abundance counts to be comparable among studies [@callahan2016dada2].  
&nbsp;  
First, sequences were trimmed following strict quality thresholds (removing primers and low quality nucleotides, see parameter details in the accompanying `R` scripts). Following this, we applied the error model algorithm of `DADA2`, which incorporates quality information after filtering, unlike other OTU based methods. Then dereplication, sample inference, merging of paired end reads and removal of chimera were performed in order to obtain a sequence (ASV) table of abundance per sample. Taxonomy was assigned through the `DADA2` pipeline using the Ribosomal Database Project (RDP) Naive Bayesian Classifier algorithm from Wang *et al.* [-@wang2007naive]. Depending on support (minimum bootstrap support of 80), we assigned taxonomy from Kingdom to species. We used the silva database formatted for `DADA2` to infer bacterial taxa [@silva]. We used the Unite [@UNITE2017] fasta release (including singletons) to infer fungal taxa after formatting it to the `DADA2` format using a custom `R` script. The pipeline was run on a multithreaded (48 CPUs) computer infrastructure provided by Westgrid (https://www.westgrid.ca/support/systems/cedar) and Compute Canada (www.computecanada.ca). Note that the pipeline was run separately for fungal-root, fungal-soil, bacteria-soil and bacteria-root samples given that these were sequenced separately and therefore a specific error model for each dataset was calculated.
&nbsp;  
&nbsp;  
*Statistical analyses - plant productivity*  
Each plant species (tomato and pepper) were analyzed separately. We tested for the amendment effect (tomato: hen manure + ANE, pepper: ANE) on six plant productivity measures (fruit number, average fruit weight, shoots fresh weight, roots fresh weight, shoots dry weight, roots dry weight). We used Linear Mixed effect Models (LMM) in the R package `NLME` [@pinheiro2017nlme], which are more appropriate than an Analysis of Variance (ANOVA) given the current block design (blocks and replicates were treated as random variables). All six plant productivity measures were either square root or log transformed in order to help satisfy the assumption of normality and homogeneity of the variance of the residuals in the LMM statistical framework. For the variables *fruit number* and *average fruit weight*, we also verified statistical significance using a permutation-based 2-way ANOVA [@anderson1999empirical] given that the residuals of the LMM were not normally distributed. Results were similar according to the 2-way ANOVA.      
&nbsp;  
*Statistical analyses - microbial and fungal diversity*  
For each fungal-root, fungal-soil, bacterial-root and bacterial-soil datasets, we removed samples that showed poor sequencing output and contained few ASV. In addition, for bacterial root and soil datasets, we removed ASV that were taxonomically assigned to *mitochondria* or *chloroplast* given that these were likely sequences from the plants themselves. To remove low quality samples, we first summed the abundance of all ASV for each sample ($\sum_{i=1}^n ASV$) and eliminated samples that had fewer that a summed abundance of 1,000. In addition, we removed ASV from our dataset that were present in fewer than 5% of the samples (less than ten individuals in the soil samples or less than five in the root samples). This was done to remove very rare ASV unique to a block or replicate, but not found in the majority of samples.  
&nbsp;    
We then conducted community-based analyses looking at the amendment effect on ASV abundance in the tomato and pepper experiments separately. To visualize communities and reduce the complexity of the datasets, relative abundance of all taxa was calculated per family using the `R` package `DPLYR` [@wickham2015dplyr] and barplots were drawn using `GGPLOT2` [@wickham2016ggplot2]. ASV alpha ($a$)-diversity was calculated based on all ASV (excluding rare ASV, see paragraph above) for each sample using the inverse Simpson diversity index in `VEGAN` [@oksanen2013package]. The effect of the amendment and planting for soil communities were assessed using a Linear Mixed effect Model (LMM) model in the R package `NLME` [@pinheiro2017nlme], given the unbalanced, replicated block design. Alpha diversity was *log* transformed in order to help satisfy the assumption of normality of the residuals in the LMM statistical framework.   
&nbsp;    
Using the community matrix data of ASV abundance, we performed PERmutational Multivariate ANalysis Of VAriance tests [PERMANOVA; @anderson2001new] to identify relationships between the communities according to the experimental design. Data were analyzed separately for fungal-root, fungal-soil, bacterial-root and bacterial-soil in tomatoes and peppers. The ASV abundance matrix was Hellinger-transformed and significance was assessed using 10,000 permutations in `vegan` [@oksanen2013package]. Blocks and replicates were factored as strata in the model.  
&nbsp;    
We also performed redundancy analyses (RDAs) using the Hellinger-transformed ASV abundance matrix in `vegan` [@oksanen2013package] to visually assess the grouping of samples, ASV and their association with productivity variables (*species* scaling based on ASV matrix). Data were analyzed separately for fungal-root, fungal-soil, bacterial-root and bacterial-soil in tomatoes and peppers, giving a total of eight RDAs. Statistical significance of the RDAs were tested using an ANOVA like permutation test (10,000 permutations) in `VEGAN`. Data were constrained based on four productivity measures (fruit number, average fruits weight, shoots fresh weight, roots fresh weight). We excluded the shoots & roots dry weights as constraints to simplify the model. In addition, these were highly collinear with the fresh weight already included as constraints ($r^2$=0.98 and 0.76 for shoot dry/fresh weights and root dry/fresh weights, respectively).      
&nbsp;    
Finally, we attempted to identify candidate ASV positively associated with productivity. As such, we identified the ten ASV most positively associated with the measures of fruit number, shoots fresh weight and roots fresh weight from each redundancy analysis for a total of 40 fungal and 40 bacterial candidate ASV. We aligned candidate sequences from these candidates ASV using the Bioconductor `R` package `DECIPHER` [@wright2016using] and build pairwise distances matrices using a JC69 substitution models of DNA sequence evolution [equal base frequencies, @jukes1969evolution] in `PHANGORN` [@schliep2010phangorn]. Phylogenetic trees (neighbour-joining) for bacteria and fungi were plotted using `APE` [@paradis2004ape]. This permitted to identify if similar candidate ASV were found under different experimental conditions (soil/root, pepper/tomato), thus reinforcing their role in productivity increase, and increasing the probability that they are true positives.  
&nbsp;  
  
\newpage  

# RESULTS  
*Effects of the amendment treatment on productivity*  
The effects of the amendment treatment on tomato (hen manure + ANE) and pepper (ANE) were determined by measuring six agronomic parameters (fruit number, average fruit weight, shoots fresh weight, shoots dry weight, roots fresh weight, roots dry weight). We observed a significant increase of almost all these agronomic parameters (LMM, *p*-value<0.005, Figure 1) for amended plants except for the average fruit fresh weight for tomato that did not differ between amended and control plants (LMM, $F_{(1,23)}$ = 1.81, *p*-value=0.19, Figure 1 and Figure S1). The amendment effect was stronger in the tomato plants (fold changes between amended and control plants shown in Figure 1), likely due to the fact that these plants were fertilized with both hen manure and ANE.      
&nbsp;  
*Amplicon Sequencing*  
A total of 2.7 million paired-end raw reads were obtained for all samples combined (976,000 for fungi-soil, 920,000 for fungi-root, 309,000 for bacteria-soil and 535,000 for bacteria-root, Table S4). On average, 47,664 paired-end reads were obtained per sample. After quality filters were applied, including removing chimeras, and paired-end reads were merged, an average of 19,690 sequences remained per sample. From 192 soil samples for fungi and bacteria, and 96 root samples for fungi and bacteria, three fungi-soil samples, 15 fungi-root samples and one bacteria-root samples were removed because they had to few reads based on our strict quality thresholds.  
&nbsp;  
The `DADA2` pipeline inferred 6,112 fungal-soil, 845 fungal-root, 9,352 bacterial-soil and 2,023 bacterial-roots ASV (Table S4). In bacteria-soil, we further removed a total of 79 ASV whose taxonomy corresponded to *mitochondria* or *chloroplast* and represented 0.1% of all sequencing reads. In bacteria-root samples, we removed a total of 284 ASV that corresponded to *mitochondria* or *chloroplast* and represented 89% of all sequencing reads. After filtering out rare ASV, we retained 413, 106, 807 and 262 ASV respectively for fungal-soil, fungal-root, bacterial-soil and bacterial-roots. These retained ASV comprised 94%, 95%, 89% and 11% of all filtered-merged sequences assigned to ASV by the `DADA2` pipeline in the fungal-soil, fungal-root, bacterial-soil and bacterial-root samples, respectively.  
&nbsp;  
*Fungal and bacterial diversity in root and soil biotopes*  
The microbial community structures of soil and root samples were analyzed and the relative abundance of their taxa was determined at the family level (Figures 2 & 3). Nectriaceae dominated the fungal communities, both in the root and soil samples, while the bacterial family Bacilaceae dominated to a lesser extent the soil samples. Bacterial root communities harbored a number of different families: Streptomycetaceae, Sphingomonadaceae, Rhizobiaceae and Pseudomonadaceae among others.   
&nbsp;  
*Local ($a$-diversity)*  
The $a$-diversity was calculated separately for each sample, under each experimental condition (fungi-soil, fungi-root, bacteria-soil and bacteria-root for both tomato and pepper, Figure 4). Linear mixed effects models showed that the $a$-diversity (Inverse Simpson Index) was significantly higher in the soil biotope that in the roots for both fungi (mean $a$-diversity soil-fungi = 2.88 vs. mean $a$-diversity root-fungi = 27.3, $F_{(1,239)}$=899.5, *p*-value<0.0001) and bacteria (mean $a$-diversity soil-bacteria = 4.7 vs. mean $a$-diversity root-bacteria = 69.2, $F_{(1,223)}$=1198.1, *p*-value<0.0001).   
&nbsp;  
In soil samples, fungal $a$-diversity was not significantly different in amended versus control plants for neither tomato ($F_{(1,66)}$=1.6, *p*-value=0.21) nor pepper ($F_{(1,69)}$=1.2, *p*-value=0.05). In root samples, fungal $a$-diversity was significantly different in amended versus control plants for tomato ($F_{(1,21)}$=10.2, *p*-value=0.004), but not pepper ($F_{(1,56)}$=3.1, *p*-value=0.10).  
&nbsp;  
In soil samples, bacterial $a$-diversity was significantly different in amended versus control plants for pepper ($F_{(1,69)}$=31.5, *p*-value<0.0001), but not tomato ($F_{(1,69)}$=1.9, *p*-value=0.17). In root samples, bacterial $a$-diversity was significantly different in amended versus control plants for tomato ($F_{(1,22)}$=39.7, *p*-value<0.0001), but not pepper ($F_{(1,4)}$=0.17, *p*-value=0.70).   
&nbsp;  
*Differences in species composition among sites*  
Using a PERMANOVA, we identified that the ANE amendment treatment had a highly significant effect on both fungal and bacterial community structures (Table 1). This effect was stronger in the root (9-30% of variance explained in the models) than in the soil (3-6% of variance explained in the models).  Planting also had a significant effect on fungal and bacterial community structures (12-24% of variance explained in the models).  
&nbsp;   
Redundancy analyses (RDAs, Figures 5 for fungi and Figure 6 for bacteria) illustrated that roots fresh weight, shoots fresh weight and fruit number responded similarly, while average fruit weight behaved differentially as noted previously (in fact nearly orthogonally to the other three parameters in most ordinations). In addition, it showed that fertilized samples clustered together and were positively correlated with increases in these four productivity measures. All RDA model tested were significant ($F_{(4,10)}$>1.4, *p*-value<0.03 for all models).     
&nbsp;      
Next, we identified, for each RDA, the ten ASV most closely related to the three constraints of the model (roots fresh weight, shoots fresh weight and fruit number). These ASV were considered as putative candidate taxa most positively impacted by increases in productivity due to the ANE amendment treatment. We further analyzed the corresponding sequences for these eighty candidate ASV (ten candidates \* eight ordinations) in two separate alignments (one for fungi and one for bacterial ASV) and their accompanying phylogenetic trees.    
&nbsp;    
In fungi, we identified a number of different ASVs positively associated with productivity (Figure S2). Notably, five different ASVs taxonomically assigned to the family Microascaceae (phylum Ascomycota) in all conditions except the pepper-root were positively associated to productivity. In addition, two ASV assigned to *Mortierella* spp (soil saprotrophs in the phylum Mucoromycota), and a cluster of six different fungal closely related ASV in tomato-soil (ASV67 & ASV132), tomato-root (ASV10, ASV1017, ASV1018, ASV1019) and pepper-soil (ASV67) were positively associated to productivity in both tomato and pepper roots. Given that no taxonomy was assigned to these sequences through the `DADA2` RDP bootstrap approach, we used a BLASTn [@altschul1997gapped] approach to identify the most closely related sequences against NCBI nr. The most closely related fungal reference sequences were from an *uncultured fungus clone* (BLASTn, 86% identity,  e-value=9e-58, sequence ID: EU517002.1). Similarly, two unknown ASV (ASV61 & ASV81) also matched an *uncultured fungus clone* (BLASTn, 94% identity,  e-value=4e-165, sequence ID: DQ900965.1). Finally, another cluster of ASVs in the pepper-root was assigned to *Olpidium brassicae*, a putative fungal parasite belonging to flagellate fungi [@lay2018taxonomy].   
&nbsp;    
In bacteria-roots, we identified a large diversity of ASV positively correlated (increased abundance of these ASV) with productivity (Figure S3), Among others we identified *Rhizobium*, *Sphingomonas*, *Sphingobium*, *Bradyrhizobium* in both the soil and root biotopes and tomato and pepper species. 
\newpage  
  
# DISCUSSION
In the current study, we investigated the effects of *Ascophyllum nodosum* extracts (ANE) on root, shoot and fruit biomass in addition to bacterial and fungal communities in tomato and pepper. Overall parameters related to plant growth (root, shoot and fruit weights) significantly increased in both plant species in response to amendment treatment. These results corroborate previous studies documenting the impact of ANE on productivity in strawberries [@alam2013effect] and carrots [@Alam2014].  
&nbsp;    
In the tomato experimental set up, the effect of the amendment treatment was especially high, likely due to the fact that plants were also fertilized with hen manure in addition to ANE (Figure 1). This was not the case for the pepper plants and the increase in productivity was solely due to the addition of ANE. The commercial extract used in this investigation contained about 0.1% nitrogen, 0.2% phosphorus, 5% potassium, along with several micronutrients (Table S3). In the current experimental setup, ANE was diluted to 3.5 ml/L prior to application (250 ml per tray every two weeks). In fact, in the tomato plants the amounts of N, P and K supplied via the application of ANE were 200-1000 times less than from the hen manure itself. As such, these nutrients were given at very low concentrations relative to the crop requirements and are not expected to significantly impact growth relative to a regular agricultural fertility program [@bruulsema2012fertilizing;@alam2013effect]. Instead, organic molecules such as betaines, polyamines, cytokinins, auxins, oligosaccharides, amino acids and vitamins present in ANE have been found to have overall beneficial productivity effects on plant growth [@khan2009seaweed;@Craigie2010;@craigie2011seaweed;@Battacharyya2015].  
&nbsp;    
One of primary goal of the study was to document how bacterial and fungal communities responded to the addition of ANE. We used Illumina MiSeq amplicon sequencing targeting DNA regions specific to fungi (ITS) and bacteria (16S). Then, we identified bacterial and fungal taxa present in the samples using a relatively novel bioinformatics approach developed by @callahan2016dada2. The approach, based on the widely used programming language `R` [@team2018r], identifies unique, non-clustered sequences (ASV) that are then comparable among studies. In addition, the current analytical pipeline uses a bayesian classifier for taxonomy rather than the widely used BLAST approach, thus providing more conservative, but more accurate taxonomic identifications [@wang2007naive].  
&nbsp;    
In the current experimental set up, most ASV identified were rare and unique to one or a few sample. In fact, approximately 90% of all ASV were discarded given that they were found in singletons or present in very few samples and were thus not representative of a particular experimental treatment. These ‘rare’ ASV comprised a small minority of all sequencing reads (approximately 5% of all sequences), a pattern reminiscent of the early species abundance models showing that in most ecological communities, few species are exceptionally abundant whereas most are rare [@fisher1943relation]. In addition, a large fraction of the sequencing reads in the root bacterial communities likely originated from the plants themselves (identified as *chloroplast* or *mitochondria*). This may be partly explained by the fact that most of root biomass collected was from large roots (Fig. S1B), rather than fine root hair where most microbial biological activity likely takes place [@pregitzer2002fine] and therefore the total biomass extracted in the current study likely consisted of more root cell rather than biologically active bacterial biomass.  
&nbsp;  
Nectriaceae, a family of fungi in the order Hypocreales and often encountered as saprotrophs on decaying organic matter comprised most of the diversity both in the soil and plant roots (between 25-70% of the total number of sequencing reads, Figure 2). With respect to bacterial communities of the soil, these were much more diverse and comprised many different families (Figure 3). The amendment treatment had a significant effect on both fungal and bacterial $a$-diversity (total richness of ASV) in the root biotope, except for bacteria in pepper plants. In the soil biotope, it only had a significant effect for bacteria in the tomato plants (Figure 4).  
&nbsp;   
The amendment treatment significantly influenced fungal and bacterial community composition ($b$-diversity) among root and soil biotopes. This effect was relatively small (3-33% of variance explained in the models, Table 1) but significant, implying that the addition of ANE (pepper) or ANE and hen manure (tomato) is, at least partly, responsible for shaping microbial communities. In addition, a significant proportion of the variance in soil communities (12-24%) was explained by the planting effect, showing how plants can alter their microbiome. Finally, we also tested the effect of plant species identity on community structure on a combined dataset comprised of both the tomato and pepper plants. In the root biotope, we find that this effect (26 and 20% of variance explained in the models for fungal and bacterial communities, respectively, Table S5) is in line with numerous studies reporting how plants select their microbial communities [@chaparro2014rhizosphere;@reinhold2015roots]. Nevertheless, we recognize that the current experimental setup precludes any strong conclusion regarding the plant species' effect of community structure, as it does not allow to explicitly disentangle the species effect from the "addition of hen manure" effect.   
&nbsp;   
Finally, we also aimed to identify candidate taxa positively correlated with increased plant productivity in response to ANE application. In fungi, one cluster of ASV taxonomically assigned to *Mortierella* (soil saprotrophs in the phylum Mucoromycota) was positively correlated to productivity in both tomato and pepper roots. Interestingly, Li *et al.* [-@li2018mortierella] found that a closely related species (*M. elongata*) can improve soil health and stimulate production of plant growth hormones. In their study, Chung *et al.* [-@chung2007plant] showed how increases in productivity led to greater microbial biomass and greater number of saprophytic and arbuscular mycorrhizal fungi. Perhaps, this is explained by the fact that an increase in plant productivity can lead to greater substrate availability, potentially increasing the activity of saprophytic fungi feeding on this organic matter substrate.  
&nbsp;   
Several other fungal ASV in tomato (soil) and pepper (root) were linked to increases in productivity. Surprisingly, a putative plant pathogenic fungus (*Olpidium brassicaceae*, Figure S2) was positively associated with increased productivity. However, *O. brassicae* has been correlated with a decrease plant growth, but only when present in large amount [@lay2018taxonomy]. In addition, *O. brassicae* is likely a species complex and has been shown to constitute a large proportion of the plant roots or rhizosphere fungal community in many different systems, particularly in Brassicaceae crops [@lay2018taxonomy]. 
&nbsp;     
In bacteria roots samples, a diverse number of ASV were positively impacted by the amendment treatment (Figure S3) and many of those are known to be present in the root endosphere [e.g. *Rhizobium*, *Sphingomonas*, *Sphingobium*, *Bradyrhizobium* spp, @tkacz2015role]. For example *Rhizobium*, and *Bradyrhizobium* spp. have been shown to promote plant growth, P solubilization, N fixation and overall productivity in both legume and non-legumes species such as radishes [@avis2008multifaceted;@antoun1998potential].  
&nbsp;  
It is now well established that seaweed extracts have a positive effect on agricultural plant productivity. Concurrently, DNA barcoding permits a more comprehensive understanding of the diversity and ecology of microbial organisms and how they interact. In fact, plants and microbes should likely be redefined as *holobionts*, an assemblage of different species that forms an ecological unit [@margulis1991symbiosis]. In this study, we showed that the addition of ANE increased plant productivity. It also increased, by a small, but significant margin, the fungal and bacterial (only in the rhizosphere) biodiversity and changed the microbial community structure in the roots and in the rhizosphere of both tomato and pepper plants. Finally, we identified bacterial and fungal taxa, especially saprotroph, that were positivity associated with plant productivity. Further studies, for example using inoculum of the candidate microbial species linked to increases in productivity that we identified, may help to identify a causative link between liquid seaweed extracts, microbes and productivity.  
&nbsp;    

# ACKNOWLEDGMENTS   
We thank Mengxuan Kong for technical assistance in setting up the greenhouse experiment and measuring productivity; Mulan Dai for performing preliminary microbiome analysis and Simon Morvan for discussion about bioinformatics analyses and seaweed extracts. Research funding was provided by the Quebec Centre for Biodiversity Science (Fonds de recherche du Québec - Nature et Technologies, [FRQNT]) to SR, FRQNT to JM and the Natural Sciences and Engineering Research Council of Canada (NSERC) to MH. The authors declare that they have received in-kind contribution from Acadian Seaplants Ltd in the form of seaweed extract and technical support along with cash contribution from NSERC. The authors SB, JA and MH are not employed nor consultants for Acadian Seaplants Ltd and they are not patenting any of the results presented in this study. The authors confirm that Acadian Seaplants Ltd did not influence the conclusions of the study.

\newpage  
  
# REFERENCES
<div id="refs"></div>

\newpage  


```{r table1, echo=F, eval =T}
library(knitr)
#Table 1
permanova.summary = t(read.table("../results/permanova.summary_PT"))
rownames(permanova.summary) = c("fungi-soil (tomato)","fungi-soil (pepper)","fungi-root (tomato)","fungi-root (pepper)","bacteria-soil (tomato)","bacteria-soil (pepper)","bacteria-root (tomato)","bacteria-root (pepper)")
permanova.summary = permanova.summary[c(1,3,5,7,2,4,6,8),]
colnames(permanova.summary) = c("amendment","planting","amendment:planting")
kable(permanova.summary,digits =4,caption = "Variance explained by the terms in the PERMANOVA models")
``` 


$r^2$ (percentage of variance explained by the term in the model); *\*p-value<0.05, \*\*<0.005,  \*\*\*<0.0005*  
 
\newpage 
![Figure 1](../figures/Figure_1_productivity.pdf){width=600px}  
**Figure 1: Measures of plant productivity. *a* and *b* subscripts above boxplots denote significant differences (*p*-value < 0.005) according to the amendment effect (tomato: hen manure + ANE, pepper: ANE). Fold changes between the mean of the control and amended plants were also noted for significant differences (for pepper and tomato separately).**  
&nbsp;  
&nbsp;  
&nbsp;  
![Figure 2](../figures/Figure2_FAMILY_barplots_fungi.pdf){width=700px}  
**Figure 2: Barplots of the relative abundance of fungal ASV for fungi**  
&nbsp;
&nbsp;  
&nbsp;  
![Figure 3](../figures/Figure3_FAMILY_barplots_bacteria.pdf){width=700px}  
**Figure 3: Barplots of the relative abundance of bacterial ASV for bacteria**  
&nbsp;
&nbsp;  
&nbsp;  
![Figure 4](../figures/Figure4_alpha.pdf){width=600px}  
**Figure 4: Boxplot of $a$-diversity according to the amendment and planting effect for fungal-root, fungal-soil, bacteria-soil and bacteria-root for tomato and pepper. *a* and *b* subscripts above boxplots denote significant differences (*p*-value < 0.05).**  
&nbsp;  
&nbsp;  
&nbsp;  
![Figure 5](../figures/fungi/Figure5f_rda.pdf){width=600px, dpi=700}  
**Figure 5: Redundancy analyses (RDA) for fungal ASV (species scaling). Labeled samples were colored in gray (unfertilized) or dark yellow (fertilized). Red *+* signs represent individual ASV, while blue *+* signs are the ten ASV most closely associated with the three productivity measures of root fresh weight, shoots fresh weight and fruit number. Blue arrows are the four productivity measures used as constraints in the ordinations.** 
&nbsp;  
&nbsp;  
&nbsp;  
![Figure 6](../figures/bacteria/Figure6b_rda.pdf){width=600px, dpi=700}  
**Figure 6: Redundancy analyses (RDA) for bacterial ASV (species scaling). Labeled samples were colored in gray (unfertilized) or dark yellow (fertilized). Red *+* signs represent individual ASV, while blue *+* signs represent the ten ASV most closely associated with the three productivity measures of root fresh weight, shoots fresh weight and fruit number. Blue arrows are the four productivity measures used as constraints in the ordinations.** 
&nbsp;  
